src/main/java/core/basesyntax/process/FruitDataProcessImpl.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.process;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitDataProcessImpl implements FruitDataProcess {
+    private Map<String, Integer> fruitsQuantity;

```

Comment:
Create Storage and add this map to it, 
making it public static final Map<String, Integer> fruits = new HashMap<>();